Wrappers/Button.cs

```diff
@@ -0,0 +1,100 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using System.Collections.ObjectModel;
+using System.Xml.Linq;
+using Utilities;
+
+namespace Wrappers
+{
+    public class Button : WebPageElement
+    {
+        public Button(IWebElement element) : base(element)
+        {
+        }
+
+        public Button()
+        {
+        }
+
+        public void ClickWhenReady(IWebDriver driver,IWebElement element)
```

Comment:
Why are you accept some external `element` as a parameter? You method should work with an instance that calls it. So you must use your property `Element`.

Also remove `driver` parameter too, use `Driver` property. Check everywhere.

Reply:
It works, changed